### PR TITLE
12578 - Process Set Piece Property expressions correctly

### PR DESCRIPTION
**Pull request by BrentEaston**

### CODE REVIEW

**Comment by BrentEaston** [comment]
Tested successfully by Mark

File: vassal-app/src/main/java/VASSAL/configure/BeanShellExpressionConfigurer.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ public Component getControls() {
       validator = new Validator();
       nameField = new JTextArea(1, 100);
 
-      nameField.setFont(new Font("Monospaced", Font.PLAIN, 14));
+      nameField.setFont(new Font("courier new", Font.PLAIN, 14));
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/configure/BeanShellExpressionConfigurer.java:213]: "courier new" isn't guaranteed to exist. What happens when it doesn't?

Comment (uckelman) [reply vassal-app/src/main/java/VASSAL/configure/BeanShellExpressionConfigurer.java:213]: Also, why have this differ from the monospaced fonts everywhere else?

Comment (BrentEaston) [reply vassal-app/src/main/java/VASSAL/configure/BeanShellExpressionConfigurer.java]: oops, that was a test to confirm that on my system, Courier New was in fact the selected monospaced on windows. Fixed.

**Pull request merged**